docs/kyma/docs/032-gs-gke-installation.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Cluster Kyma installation
+type: Getting Started
+---
+
+## Prepare own kyma-installer image
+
+You can skip this step if you want to install image created from unmodified sources. 
+
+1. Checkout kyma-project and enter root folder.
+2. Build image that will include current installation and resources charts and is based on current installer image (you can find it in the installation/resources/installer.yaml file)
+    ```
+    docker build -t kyma-installer:latest -f kyma-installer/kyma.Dockerfile . --build-arg INSTALLER_VERSION=63484523
+    ```
+3. Push image to docker hub:
+    ```
+    docker tag kyma-installer:latest [YOUR_DOCKER_LOGIN]/kyma-installer:latest
+
+
+## DNS setup
+
+1. Get some domain and delegate it to google. You can get free domain from freenom.com site. I reserved kyma.ga domain.
+
+2. Create DNS managed zone in your google project. You can create if from console.cloud.google.com. Navigate to Network Services, Cloud DNS and select Create Zone. Command line version:    
+    ```
+    gcloud dns --project=$PROJECT managed-zones create kymaga-zone --description= --dns-name=kyma.ga.
```

Comment:
The env `PROJECT` is used here, and an introduction of it is later. It should be explained at the beginning what it is.

Reply:
`kymaga-zone` should be changed. We need to highlight parameter which are required.